apis/v1/httproute_types.go

```diff
@@ -1153,6 +1153,34 @@ type HTTPRequestMirrorFilter struct {
 	//
 	// Support: Implementation-specific for any other resource
 	BackendRef BackendObjectReference `json:"backendRef"`
+
```

Comment:
The type containing these two fields `HTTPRequestMirrorFilter` should have CEL validation that ensures that a maximum of one type is non-empty. Recommend adding some simple corresponding tests in https://github.com/kubernetes-sigs/gateway-api/blob/main/pkg/test/cel/httproute_experimental_test.go.